tests/acceptance/advanced-test.js

```diff
@@ -0,0 +1,63 @@
+import moduleForAcceptance from '../helpers/module-for-acceptance';
```

Comment:
@Aaronfischer The name of this test is too broad. Let's be more specific. Such a `relationships-test.js`